lib/widgets/witnet/transactions/value_transfer/create_dialog_box/vtt_builder/01_recipient_step.dart

```diff
@@ -229,12 +193,9 @@ class RecipientStepState extends State<RecipientStep>
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
     final extendedTheme = theme.extension<ExtendedTheme>();
-    _addressFocusNode.addListener(() => _validateAddress());
-    _amountFocusNode.addListener(() => _validateAmount());
-    _minerFeeFocusNode.addListener(() => _validateFee());
     return Form(
       key: _formKey,
-      autovalidateMode: AutovalidateMode.onUserInteraction,
+      autovalidateMode: AutovalidateMode.disabled,
```

Comment:
Realized this does not validate the form on clicking outside the input (on unfocus)

Reply:
fixed!